app/javascript/dashboard/routes/dashboard/settings/SettingsWrapper.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
Base component for setting page

Reply:
https://linear.app/chatwoot/issue/CW-3124/create-a-layout-component-for-sla-settings